query-planner-js/src/QueryPlan.ts

```diff
@@ -39,6 +39,12 @@ export interface FetchNode {
   operationName: string | undefined;
   operationKind: OperationTypeNode;
   operationDocumentNode?: DocumentNode;
+  inputRewrites?: FetchDataRewrite[];
+}
+
+export interface FetchDataRewrite {
```

Comment:
Added some comments/doc (and ever so slightly refactored; the reason is that I extend this in the patch at #2294, and doing so I noticed that this could done a bit cleaner so changed to that here).